From: Alexander Graf <agraf@suse.de>
Date: Wed, 20 Mar 2013 21:17:25 +0100
Subject: [PATCH] SMDKV310: Fix compilation
Patch-Mainline: No

Apparently __devinitdata has gone from the kernel tree. Remove it from
the regulator struct definition too, so that the code compiles.

Signed-off-by: Alexander Graf <agraf@suse.de>

Index: linux-3.9-rc3-master/arch/arm/mach-exynos/mach-smdkv310.c
===================================================================
--- linux-3.9-rc3-master.orig/arch/arm/mach-exynos/mach-smdkv310.c
+++ linux-3.9-rc3-master/arch/arm/mach-exynos/mach-smdkv310.c
@@ -382,7 +382,7 @@ static void __init smdkv310_reserve(void
 	s5p_mfc_reserve_mem(0x43000000, 8 << 20, 0x51000000, 8 << 20);
 }
 
-static struct regulator_consumer_supply vddmmc_consumers[] __devinitdata = {
+static struct regulator_consumer_supply vddmmc_consumers[] = {
 	REGULATOR_SUPPLY("vmmc", "s3c-sdhci.0"),
 	REGULATOR_SUPPLY("vmmc", "s3c-sdhci.1"),
 	REGULATOR_SUPPLY("vmmc", "s3c-sdhci.2"),
